### PR TITLE
fix: [pipe-23087]: dont have disable on create pr

### PR DESCRIPTION
**Pull request by cjlee01**
dont have disable on create pr and pick pr title from commits

### CODE REVIEW

File: packages/playground/src/layouts/SandboxPullRequestCompareLayout.tsx [hunk 1]
```diff
@@ -85,6 +86,15 @@ const SandboxPullRequestCompare: React.FC<SandboxPullRequestCompareProps> = ({
     }
   })
 
+  useEffect(() => {
+    if (commitData && commitData.length > 0) {
+      reset({
+        title: commitData[commitData.length - 1].title,
```
Comment (vardanbansal-harness) [inline packages/playground/src/layouts/SandboxPullRequestCompareLayout.tsx:92]: Is `title` optional? if yes, lets use optional chaining `?.title`

File: packages/playground/src/pages/sandbox-pull-request-conversation-page.tsx [hunk 2]
```diff
@@ -228,12 +232,13 @@ export default function SandboxPullRequestConversationPage() {
         <SandboxLayout.Column>
           <SandboxLayout.Content className="pl-0 pr-0">
             <PullRequestSideBar
+              addReviewers={noop}
+              usersList={undefined}
               // repoMetadata={undefined}
-              pullRequestMetadata={undefined}
+              pullRequestMetadata={{ source_sha: pullReqMetadata?.source_sha as string }}
```
Comment (vardanbansal-harness) [inline packages/playground/src/pages/sandbox-pull-request-conversation-page.tsx:238]: Any reason to typecast? `sha` should actually just be string?

**Review by vardanbansal-harness — APPROVED** [review]

File: packages/playground/src/pages/sandbox-pull-request-conversation-page.tsx (hunk 2 above)
Comment (cjlee01) [reply packages/playground/src/pages/sandbox-pull-request-conversation-page.tsx]: removed

File: packages/playground/src/layouts/SandboxPullRequestCompareLayout.tsx (hunk 1 above)
Comment (cjlee01) [reply packages/playground/src/layouts/SandboxPullRequestCompareLayout.tsx]: fixed

**Pull request merged**